package.json

```diff
@@ -68,7 +68,7 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^1.8.0",
-    "@edx/paragon": ">= ^7.0.0 < 13.0.0",
+    "@edx/paragon": ">= ^7.0.0",
```

Comment:
Updated this constraint because its raising conflict errors when running `npm update` in other packages using frontend-component-header. I've tested that tests ran successfully even after updating the constraint.

@openedx/community-engineering please let me know if you think it may raise any other issues that I'm unaware of.